seqr/models.py

```diff
@@ -222,26 +225,6 @@ def save(self, *args, **kwargs):
             user = self.created_by
             user.groups.add(self.can_edit_group, self.can_view_group)
 
-    def delete(self, *args, **kwargs):
-        """Override the delete method to also delete the project-specific user groups"""
-
-        super(Project, self).delete(*args, **kwargs)
-
-        self.can_edit_group.delete()
-        self.can_view_group.delete()
```

Comment:
the database migration changes the `on_delete` mode to cascade so they will automatically be deleted instead of us needing to maintain this custom logic

Reply:
I was confused about how this works. I will add this back